<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPreferences</class>
 <widget class="QDialog" name="DialogPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="Настройки">Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="flowControlLabel_4">
       <property name="text">
        <string extracomment="Язык:">Language:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string extracomment="Количество байт в строке шестнадцатиричного отображения">bytes in row of Hex-browser</string>
       </property>
       <property name="whatsThis">
        <string extracomment="Количество байт в строке шестнадцатиричного отображения">bytes in row of Hex-browser</string>
       </property>
       <property name="text">
        <string extracomment="байт в строке Hex-браузера">bytes in row of Hex-browser</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="flowControlLabel_3">
       <property name="text">
        <string>Codec:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboLanguage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string notr="true">ru</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">en</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSpinBox" name="sbBytesPerLine">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>bytes in row of Hex-browser</string>
       </property>
       <property name="whatsThis">
        <string>bytes in row of Hex-browser</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboCodec">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSpinBox" name="sbFontSize">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>UI font size in points</string>
       </property>
       <property name="whatsThis">
        <string>UI font size in points</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string extracomment="Количество байт в строке шестнадцатиричного отображения">UI font size in points</string>
       </property>
       <property name="whatsThis">
        <string extracomment="Количество байт в строке шестнадцатиричного отображения">UI font size in points</string>
       </property>
       <property name="text">
        <string extracomment="байт в строке Hex-браузера">Font size</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbSavePos">
     <property name="toolTip">
      <string>Save position and size of application on exit</string>
     </property>
     <property name="whatsThis">
      <string>Save position and size of application on exit</string>
     </property>
     <property name="text">
      <string extracomment="Сохранять позицию при выходе">Save position on exit</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string notr="true">OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
